quizzes/10_class.test.js

```diff
@@ -0,0 +1,72 @@
+const p = new Pixel(5, 12);
+const q = Pixel.fromObj({x: 0, y: 0});
+
+console.log(
+  // p.distance(q)
+  // q.distance(p)
+  // p.midpoint(q)
+  // q.midpoint(p)
+  //
+)
+
+const v = new Voxel(5, 6, 7);
+const w = Voxel.fromObj({x: 2, y: 3.4, z: 7.9});
+
+console.log(
+  // v.distance(w)
+  // w.midpoint(v)
+)
+
+
+// TODO: refactor into class syntax
+function Pixel(x=0, y=0) {
```

Comment:
I think we'll need to move the `console.log` to the bottom because the `class` wont get hoisted like a function declaration does.